benches/large_case.rs

```diff
@@ -5,15 +5,18 @@ extern crate criterion;
 use self::criterion::*;
 
 use pubgrub::package::Package;
+use pubgrub::range::Range;
 use pubgrub::solver::{resolve, OfflineDependencyProvider};
-use pubgrub::version::{NumberVersion, SemanticVersion, Version};
+use pubgrub::version::{NumberVersion, SemanticVersion};
+use pubgrub::version_set::VersionSet;
 use serde::de::Deserialize;
-use std::hash::Hash;
 
-fn bench<'a, P: Package + Deserialize<'a>, V: Version + Hash + Deserialize<'a>>(
+fn bench<'a, P: Package + Deserialize<'a>, V: VersionSet + Deserialize<'a>>(
```

Comment:
Can we call it `VS:` not `V:`, I think that will be more consistent with our usage in the rest of the package?
Then the `where` can become `where VS::V: Deserialize<'a>`.

Reply:
Done